addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -391,7 +391,8 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
         //display rainbow at the end of any tour
         if (this.tours[tour_name].rainbowMan && this.running_tour !== tour_name &&
             this.tours[tour_name].current_step === this.tours[tour_name].steps.length) {
-            const message = this.tours[tour_name].rainbowManMessage ||
+            const rainbowManMessage = this.tours[tour_name].rainbowManMessage
+            const message = typeof rainbowManMessage === 'function' ? rainbowManMessage() : rainbowManMessage ||
                 _t('<strong><b>Good job!</b> You went through all steps of this tour.</strong>');
```

Comment:
```suggestion
            let message = this.tours[tour_name].rainbowManMessage;
            if (message) {
                message = typeof message === 'function' ? message() : message;
            } else {
                message = _t('<strong><b>Good job!</b> You went through all steps of this tour.</strong>'); 
            }    
```

Reply:
I know it's not yet documented, but it would be nice if you could add rainbowManMessage in the docstring of `register`